diff --git a/third_party/ffmpeg/libavcodec/x86/vp3dsp_init.c b/third_party/ffmpeg/libavcodec/x86/vp3dsp_init.c
index cc3eba4..0721482 100644
--- a/third_party/ffmpeg/libavcodec/x86/vp3dsp_init.c
+++ b/third_party/ffmpeg/libavcodec/x86/vp3dsp_init.c
@@ -66,6 +66,7 @@ void ff_vp3_h_loop_filter_mmxext(uint8_t *src, int stride,
 #if HAVE_6REGS
 static void put_vp_no_rnd_pixels8_l2_mmx(uint8_t *dst, const uint8_t *a, const uint8_t *b, ptrdiff_t stride, int h)
 {
+/*
 //    START_TIMER
     MOVQ_BFE(mm6);
     __asm__ volatile(
@@ -94,6 +95,7 @@ static void put_vp_no_rnd_pixels8_l2_mmx(uint8_t *dst, const uint8_t *a, const u
         :"r"((x86_reg)stride), "r"((x86_reg)3L*stride)
         :"memory");
 //    STOP_TIMER("put_vp_no_rnd_pixels8_l2_mmx")
+*/
 }
 #endif /*HAVE_6REGS */
 #endif /* HAVE_MMX_INLINE */
